docker/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+# we have a really simple setup here
+# we use docker-compose only as a convenient way to specify docker build parameters via a yaml file
+# you could as well use Dockerfile directly with `docker build` and config passed via command-line args
+#
+# tips:
+#   - to run lndmanage from docker, you can use our wrapper script ./lndmanage.sh
+#   - see other scripts in this folder, also check the docker section in the main readme
+
+version: '3.7'
```

Comment:
Would it be problematic to lower the version? From the Ubuntu standard installation, I can run version 3.5.

Reply:
Sure, feel free to downgrade it. It should work the same I believe.